blueocean-web/src/main/js/main.jsx

```diff
@@ -158,9 +161,11 @@ function startApp(routes, stores) {
 
     // Start React
     render(
-        <Provider store={store}>
-            <Router history={history}>{ makeRoutes(routes) }</Router>
-        </Provider>
+          <I18nextProvider i18n={ i18n }>
```

Comment:
that is the react enabler see above comment for non-react components. They would use i18n service directly.

You asked about blueocean-web/src/main/js/i18n.js that is a simple configure factory for the i18n service this can be used in a non-react context.